src/components/ItaliaTheme/View/EventoView.jsx

```diff
@@ -304,24 +307,22 @@ const EventoView = ({ content, location }) => {
                     )}
                   </OfficeCard>
                 ))}
-
-                {content?.evento_supportato_da?.length > 0 && (
-                  <>
-                    <h5 className="mt-4 supported-by">
-                      {intl.formatMessage(messages.supported_by)}
-                    </h5>
-                    {content?.evento_supportato_da?.map((item) => (
-                      <OfficeCard
-                        key={item['@id']}
-                        office={item}
-                        extended={true}
-                        icon={'it-pa'}
-                      />
-                    ))}
-                  </>
-                )}
               </article>
             ) : null}
+
+            {content?.evento_supportato_da?.length > 0 && (
```

Comment:
questo non andrebbe messo sotto la sezione 'contatti' (interni o esterni)? 
Se non metto contatti ma metto qualcosa in evento_supportato_da, mi vedo questa mini capitoletto sparato li senza un padre. 
Secondo me andrebbe mostrato dentro al titolo 'contatti' come dice agid nel documento https://docs.google.com/spreadsheets/d/1bE0Ns0LsU0VDvCBT1WXZ5_yIxJU5AbUYcu_F8yMfpHQ/edit#gid=311121645&range=A42

Reply:
Ho aggiunto un controllo che se non sono presenti contatti interni o esterni e quindi solo "evento_supportato_da" viene mostrato il titolo "Contatti" con dentro appunto "evento_supportato_da"